corehq/apps/domain/forms.py

```diff
@@ -1485,6 +1490,14 @@ def process_subscription_management(self):
         new_plan_version = DefaultProductPlan.get_default_plan_by_domain(
             self.domain, edition=self.cleaned_data['software_plan_edition'],
         )
+        kwargs = {
+            'auto_generate_credits': True,
+            'date_end': self.cleaned_data['end_date'],
+            'do_not_invoice': False,
+            'internal_change': True,
```

Comment:
Yes, good catch. I think this might be the manifestation of http://manage.dimagi.com/default.asp?170773

The internal_change flag should be set for all forms that are only internal changes... so, yes - for all subclasses of `InternalSubscriptionMangementForm`